docs/databases/mongo.md

```diff
@@ -4,19 +4,30 @@ A database module for [opsdroid](https://github.com/opsdroid/opsdroid) to persis
 
 ## Requirements
 
-None.
+An accessible MongoDB server with the database that you provide already created.
 
 ## Configuration
 
 ```yaml
 databases:
   mongo:
-    host:       "my_host"     # (optional) default "localhost"
-    port:       "12345"       # (optional) default "27017"
-    database:   "my_database" # (optional) default "opsdroid"
-    user:       "my_user"     # (optional)
-    password:   "pwd123!"     # (optional)
+    host:                       "my_host"       # (optional) default "localhost"
+    port:                       "12345"         # (optional) default "27017"
+    database:                   "my_database"   # (optional) default "opsdroid"
+    default_collection_name:    "my_collection" # (optional) default "opsdroid"
```

Comment:
Have you adjusted the code to use `default_collection_name` instead of `collection`? It looks like it is still looking for `collection`.

Reply:
Oh whoops, didn't change the docs. So `default_collection_name` was originally proposed by me, but I reconsidered once we decided to go the `contextmanager` route. I don't think it's necessary to be so verbose anymore. I'll change these docs to reflect.